scheduler/config-k8s.edn

```diff
@@ -34,7 +34,17 @@
                                :factory-fn "cook.plugins.demo-plugin/launch-factory"}}
  :hostname #config/env "COOK_HOSTNAME"
  :kubernetes {:disallowed-container-paths #{"/mnt/bad"}
-              :disallowed-var-names #{"BADVAR"}}
+              :disallowed-var-names #{"BADVAR"}
+              :init-container {:command "echo test"
+                               :image "byrnedo/alpine-curl:latest"}
+              :sandbox-fileserver {:command "fileserver"
+                                   :image "twosigma/cook-fileserver:latest"
+                                   :port 23906
+                                   :resource-requirements {:cpu-request 0.1
+                                                           :cpu-limit 0.1
+                                                           :memory-request 128.0
+                                                           :memory-limit 128.0}}
+              :wrapper-script "/bin/sh -c "}
```

Comment:
Is `:wrapper-script` the right name for this? Maybe `:launch-command` or `:init-command`?

Issues with `wrapper-script`:

1. The command doesn't have to be a script (it can be a binary)
2. It's not really a "wrapper" in the sense that the "real" script isn't wrapped _inside_, but rather passed in as an argument.

Reply:
settled on `:custom-shell`